Sources/SwifterSwift/UIKit/UIViewControllerExtensions.swift

```diff
@@ -22,6 +22,20 @@ public extension UIViewController {
 
 // MARK: - Methods
 public extension UIViewController {
+    
+    
+    /// SwifterSwift: Instantiate UIViewController from storyboard
+    ///
+    /// - Parameters:
+    ///   - storyboard: Name of the storyboard where the UIViewController is located
+    ///   - bundle: Bundle in which storyboard is located
+    ///   - identifier: UIViewController's storyboard identifier
+    /// - Returns: Custom UIViewController instantiated from storyboard
+    class func instantiate(from storyboard: String = "Main", bundle: Bundle? = nil, identifier: String? = nil) -> Self {
+        let viewControllerIdentifier = identifier ?? String(describing: self)
+        let storyboard = UIStoryboard(name: storyboard, bundle: bundle)
+        return storyboard.instantiateViewController(withIdentifier: viewControllerIdentifier) as! Self
```

Comment:
Could we throw a `preconditionFailure` here if this`storyboard.instantiateViewController(withIdentifier: viewControllerIdentifier)` method returns nil, I think will be good because we can have a message explaining the reasons why this may be nil. What do you think @guykogus?

Reply:
Yeah, I think that'll be clearer than crashing on a forced cast.

Reply:
@LucianoPAlmeida The thing is `storyboard.instantiateViewController(withIdentifier: viewControllerIdentifier)` doesn't return `Optional` and comparing the return value to `nil` will always result `false`

Reply:
You can do it like this:
```Swift
guard let viewController = storyboard.instantiateViewController(withIdentifier: viewControllerIdentifier) as? Self else {
    preconditionFailure("Unable to instantiate view controller with identifier \(viewControllerIdentifier) as type \(String(describing: Self))")
}
return viewController
```